client/src/pages/SearchPage/SearchPage.js

```diff
@@ -80,6 +80,12 @@ class SearchPage extends React.Component {
         });
     }
 
+    handleSortChange(sort) {
+        this.search({
+            sort
+        });
```

Comment:
The way you wrote `SearchPage.search()` is genius.

Reply:
The way I write everything is genius. Especially bugs. Genius bugs. We have the best bugs. The biggest bugs.